src/test/scala/chiselTests/Vec.scala

```diff
@@ -32,6 +32,15 @@ class IOTesterMod(vecSize: Int) extends Module {
   io.out := io.in
 }
 
+class OneBitUnitRegVec extends Module {
+  val io = IO(new Bundle {
+    val out = Output(UInt(1.W))
+  })
+  val oneBitUnitRegVec = Reg(Vec(1, 1.U))
```

Comment:
I think @ducky64  is correct. The second `Vec` argument should be a `UInt(1.W)`.